tests/e2e/config/jest-playwright.config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  // prettier-ignore
+  browsers: [
+    'chromium',
+    'firefox',
+    'webkit',
```

Comment:
@jhildenbiddle What OS are you in? Does `webkit` testing work in Linux? I'm in Linux. Haven't tried it yet.

Reply:
It failed for me. Looks like we need some conditionals here depending on OS.

![failed-test](https://user-images.githubusercontent.com/297678/87822965-4ed3bb00-c827-11ea-9b1e-ace472f73fb8.png)

Reply:
@trusktr -- macOS. I agree that we should test this on Linux and Windows before considering it ready to go. Let me do a little research on my end regarding Linux and Webkit testing. It will require me to get a Linux VM up and running, but that's a useful this to have anyway.

What Linux distro are you running?